installscripts/wizard/scenarios/support/jazz_common.py

```diff
@@ -75,6 +75,12 @@ def parse_and_replace_parameter_list(terraform_folder, parameter_list):
                    get_atlassian_tools_path() + "lib/bitbucket-cli-6.7.0.jar",
```

Comment:
can you add some description in the PR. It will give some context on why you are making this change

Reply:
Done